components/support/locale/src/main/java/mozilla/components/support/locale/LocaleManager.kt

```diff
@@ -48,6 +48,14 @@ object LocaleManager {
         return currentLocale
     }
 
+    /**
+     * Remove the locale code from the storage. Useful when having the option for a default (use
+     * system locale), in which case [getCurrentLocale] will return null
+     */
```

Comment:
nit: I think it will better to just mention the intended behavior without mentioning the internal details as they could change in the future.

Reply:
Changed the docs, it's now similar to the one left by you above, thanks